### ISSUE TITLE
Abyssal small nosferatus have no name

### ISSUE THREAD

**Issue by DarkFenX** [pyfa-org/Pyfa#1722]
### Detailed steps to reproduce:
Apply any mutaplasmid to any small nosf

### Expected behavior:
Small Abyssal Energy Nosferatu

### Actual behavior:
![image](https://user-images.githubusercontent.com/275209/44741538-0feace00-ab06-11e8-8e3f-a63e80efd6e0.png)

**Comment by blitzmann** [comment]
This was an issue with the previous release (CCP didn't actually have a name for the item in their files, even ESI showed a placeholder name). I manually edited the name for release. I thought they had fixed this with the next patch... :/

ninja edit: yeah, it's still bugged on CCPs side.

https://esi.evetech.net/latest/universe/types/48419/?datasource=tranquility&language=en-us

I don't actually know what happens in game when you apply mutaplasmid to a small nos. When I raised this issue to Larrikin last release cycle he said that the type was indeed in the wild in the game.

Will poke him again
